ts/src/profiler.ts

```diff
@@ -282,10 +281,11 @@ export class Profiler extends common.ServiceObject {
       }
       throw new Error(message.toString());
     }
-    if (!isRequestProfile(body)) {
-      throw new Error(`Profile not valid: ${body}.`);
+    if (!isRequestProfile(prof)) {
+      throw new Error(`Profile not valid: ${prof}.`);
     }
-    return body;
+    this.logger.debug(`Successfully created profile ${prof.profileType}`);
```

Comment:
Can you add a debug message (if there is no one, and I think there isn't) before the create profiler call is started?

Reply:
Done.